v0/destinations/marketo/util.js

```diff
@@ -154,6 +166,20 @@ const responseHandler = (destinationResponse, _dest) => {
     "during Marketo Response Handling",
```

Comment:
Can we refactor `responseHandler` and `marketoResponseHandler`? Both do the same but for different stages and only slight changes in implementation.

Reply:
Have refactored it to use only one responseHandler. Can you please check and let me know if it is ok ?